build.gradle

```diff
@@ -25,8 +25,8 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
         classpath "com.vanniktech:gradle-android-junit-jacoco-plugin:0.16.0"
         classpath "com.github.ben-manes:gradle-versions-plugin:0.36.0"
-        classpath "org.jetbrains.dokka:dokka-gradle-plugin:1.4.32"
-        classpath "org.jetbrains.dokka:android-documentation-plugin:1.4.32"
+        classpath "org.jetbrains.dokka:dokka-gradle-plugin:1.5.0"
```

Comment:
I wonder if this dokka update doesn't break publishing (it's usually pretty fragile integration). Not sure how to best test without trying to run the publish task. Maybe can try to publish to local maven.

Reply:
Ah, ok. I'm going to hold off on the dokka update then for the time being and keep the rest.